var/spack/repos/builtin/packages/gpu-burn/package.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class GpuBurn(Package, CudaPackage):
+    """Multi-GPU CUDA stress test. Note that the file pointed to by COMPARE_PTX
+    needs to be copied or linked to the current working directory before
+    running gpu_burn."""
+
+    homepage = "http://wili.cc/blog/gpu-burn.html"
+    url      = "http://wili.cc/blog/entries/gpu-burn/gpu_burn-1.0.tar.gz"
+
+    version('1.0', sha256='d55994f0bee8dabf021966dbe574ef52be1e43386faeee91318dd4ebb36aa74a')
+
+    patch('Makefile.patch')
+
+    # This package uses CudaPackage to pick up the cuda_arch variant. A side
+    # effect is that it also picks up the cuda variant, but cuda is required
+    # for gpu-burn so is not really a variant.
+    variant('cuda', 'True')
```

Comment:
This is an interesting use case that we didn't really think about when we wrote the `CudaPackage`.

Can you copy the description to the variant?